rtsp.go

```diff
@@ -184,13 +184,13 @@ type rtspCamera struct {
 	lazyDecode       bool
 	iframeOnlyDecode bool
 
-	closeMu    sync.RWMutex
-	vsConfig   *videostore.Config
-	vs         videostore.H264RTPVideoStore
-	auMu       sync.Mutex
-	au         [][]byte
-	client     *gortsplib.Client
-	rawDecoder *decoder
+	closeMu         sync.RWMutex
+	vsConfig        *videostore.Config
+	videoStoreMuxer *videoStoreMuxer
```

Comment:
I swapped out the videostore with a videoStoreMuxer 

This wraps the videostore and allows simplifying the videostore API